elasticsearch/run.sh

```diff
@@ -98,6 +98,9 @@ info "ES_JAVA_OPTS: '${ES_JAVA_OPTS}'"
 DHE_TMP_KEY_SIZE=${DHE_TMP_KEY_SIZE:-2048}
 export ES_JAVA_OPTS="${ES_JAVA_OPTS:-} -Djdk.tls.ephemeralDHKeySize=$DHE_TMP_KEY_SIZE"
 
+#LOG-334
+export ES_JAVA_OPTS="${ES_JAVA_OPTS:-} -Des.cgroups.hierarchy.override=/"
+
```

Comment:
nit, but can we move the `info "ES_JAVA_OPTS: '${ES_JAVA_OPTS}'"` on line 96 to after this line?